FHEM/00_SIGNALduino.pm

```diff
@@ -166,7 +166,7 @@ my %matchListSIGNALduino = (
      "15:SOMFY"					=> '^Ys[0-9A-F]+',
      "16:SD_WS_Maverick"		=> '^P47#[A-Fa-f0-9]+',
      "17:SD_UT"            		=> '^[P|u](30|79|81|83)#.*',	 # universal - more devices with different protocols
-     "18:FLAMINGO"            	=> '^P13#[A-Fa-f0-9]+',						## Flamingo Smoke
+     "18:FLAMINGO"            	=> '^(P13|P13.1)#[A-Fa-f0-9]+',  # Flamingo Smoke
```

Comment:
Ich glaube in der Matchlist werden keine Substitions ausgewertet und die Regex ist so auch nicht korrekt.

`.` steht für jedes beliebige Zeichen.

Damit sollte das gewünschte Verhalten eintreten:
`^P13\.?1?#[A-Fa-f0-9]+`

Details siehe hier
https://regex101.com/r/qRMNiD/1

Reply:
@sidey79 Mit diesem Commit 
https://github.com/RFD-FHEM/RFFHEM/pull/316/commits/0ede8a0a39ba425ed397e705d8ba17a3dc522f5b
sollte es angepasst sein.